datumaro/plugins/ndr.py

```diff
@@ -39,10 +39,13 @@ class NDR(Transform, CliPlugin):
     |n
     Available undersampling policies (the `-u` parameter):|n
     - `uniform` - sample data with uniform distribution|n
-    - `inverse` - sample data with reciprocal of the number of number of|n
-    |s|sitems with the same similarity|n
+    - `inverse` - sample data with reciprocal of the number of number of \
```

Comment:
```suggestion
    - `inverse` - sample data with reciprocal of the number of number of
```
This is going to spoil CLI output, probably.